com.unity.render-pipelines.universal/Shaders/BakedLit.shader

```diff
@@ -61,97 +60,12 @@ Shader "Universal Render Pipeline/Baked Lit"
-                UNITY_TRANSFER_INSTANCE_ID(input, output);
-                UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(output);
-
-                VertexPositionInputs vertexInput = GetVertexPositionInputs(input.positionOS.xyz);
-                output.vertex = vertexInput.positionCS;
-                output.uv0AndFogCoord.xy = TRANSFORM_TEX(input.uv, _BaseMap);
-                output.uv0AndFogCoord.z = ComputeFogFactor(vertexInput.positionCS.z);
-
-                // normalWS and tangentWS already normalize.
-                // this is required to avoid skewing the direction during interpolation
-                // also required for per-vertex SH evaluation
-                VertexNormalInputs normalInput = GetVertexNormalInputs(input.normalOS, input.tangentOS);
-                output.normalWS = normalInput.normalWS;
-    #if defined(_NORMALMAP)
-                real sign = input.tangentOS.w * GetOddNegativeScale();
-                output.tangentWS = half4(normalInput.tangentWS.xyz, sign);
-    #endif
-                OUTPUT_LIGHTMAP_UV(input.lightmapUV, unity_LightmapST, output.lightmapUV);
-                OUTPUT_SH(output.normalWS, output.vertexSH);
-
-                return output;
-            }
-
-            half4 frag(Varyings input) : SV_Target
-            {
-                UNITY_SETUP_INSTANCE_ID(input);
-                UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
-
-                half2 uv = input.uv0AndFogCoord.xy;
-                half4 texColor = SAMPLE_TEXTURE2D(_BaseMap, sampler_BaseMap, uv);
-                half3 color = texColor.rgb * _BaseColor.rgb;
-                half alpha = texColor.a * _BaseColor.a;
-                AlphaDiscard(alpha, _Cutoff);
-
-#ifdef _ALPHAPREMULTIPLY_ON
-                color *= alpha;
-#endif
-
-    #if defined(_NORMALMAP)
-                half3 normalTS = SampleNormal(uv, TEXTURE2D_ARGS(_BumpMap, sampler_BumpMap)).xyz;
-                float sgn = input.tangentWS.w;      // should be either +1 or -1
-                float3 bitangent = sgn * cross(input.normalWS.xyz, input.tangentWS.xyz);
-                half3 normalWS = TransformTangentToWorld(normalTS, half3x3(input.tangentWS.xyz, bitangent, input.normalWS));
-    #else
-                half3 normalWS = input.normalWS;
-    #endif
-                normalWS = NormalizeNormalPerPixel(normalWS);
-                color *= SAMPLE_GI(input.lightmapUV, input.vertexSH, normalWS);
-                #if defined(_SCREEN_SPACE_OCCLUSION)
-                    float2 normalizedScreenSpaceUV = GetNormalizedScreenSpaceUV(input.vertex);
-                    color *= SampleAmbientOcclusion(normalizedScreenSpaceUV);
-                #endif
-                color = MixFog(color, input.uv0AndFogCoord.z);
-                alpha = OutputAlpha(alpha, _Surface);
-
-                return half4(color, alpha);
-            }
+            #include "Packages/com.unity.render-pipelines.universal/Shaders/BakedLitForwardPass.hlsl"
```

Comment:
I have PR where I've done the same (similar) refactoring. There might be conflict here.

Reply:
Thanks for the warning - I'll keep my eye out for it

Reply:
Okay - thanks for letting me know, I'll keep an eye out for it. Merging "master" into this branch is always fun!